backend/lib/components/Geo/gmaps/getFacDistances.js

```diff
@@ -0,0 +1,41 @@
+const {Client} = require("@googlemaps/google-maps-services-js");
+const Place = require('./utils/place.js');
+const {
+    config: { geo },
+  } = require("../../../../config");
+
+var placesWithDistances = [];
+
+const getFacilityDistances = async (location, places) => { 
```

Comment:
Same as above.

Reply:
Moving array variable inside.

Reply:
Also, we should change any `var` to `let`, and make sure they are in the right scope.

Reply:
Ok - changed to let.